docs/pages/project/contributing/contributing-docs.md

```diff
@@ -316,24 +318,19 @@ Just for fun, here are all the types:
 {% include alert.html type="primary" content="This is a primary alert" %}
 {% include alert.html type="secondary" content="This is a secondary alert" %}
 
-
-### Alerts
-
-{% include alert.html type="info" title="What is an alert?" content="An alert is a box that can stand out to indicate important information. You can choose from levels success, warning, danger, info, and primary. This example is an info box, and the code for another might look like this:" %}
+#### alert.html
 
 ```
-{%raw%}{% include alert.html type="info" title="Here is another!" %}{%endraw%}
+The [alert.html](https://github.com/meshery/meshery/blob/master/docs/_includes/alert.html) file is used to format and display notes, warnings, and other informational callouts in Meshery. 
+It provides a consistent and visually appealing way to draw the user's attention to important information.
+```
+To use the `alert.html` feature, include the following code snippet in your documentation:
+``` 
+     <div class="alert alert-{{ include.type }}">
+    <h4 class="alert-heading">{% if include.title %}{{ include.title }}{% else %}{{ include.type }}{% endif %}</h4>
+    {{ include.content }}
+    </div>
 ```
```

Comment:
How this code will include that feature? This is just the code snippet of `alert.html` file. Also where are examples and the content that you have wrote in your previous PR, explaining every section?

Reply:
> How this code will include that feature? This is just the code snippet of `alert.html` file. Also where are examples and the content that you have wrote in your previous PR, explaining every section?

Okay, I am modifying it